management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -534,17 +534,18 @@ public Map<String, String> getReplication(@RpcParam(name = "keyspaceName") Strin
     return rows.one().getMap("replication", UTF8Type.instance, UTF8Type.instance);
   }
 
-  @Rpc(name = "getTables")
-  public List<String> getTables(@RpcParam(name = "keyspaceName") String keyspaceName) {
+  @Rpc(name = "getTables", multiRow = true)
+  public List<Table> getTables(@RpcParam(name = "keyspaceName") String keyspaceName) {
```

Comment:
If I understood the code correctly, this will serialize each object as a row, and each public fields of the object as a column.